### ISSUE TITLE
The overview page crashed: `TypeError: i is not iterable`

### ISSUE THREAD

**Issue by iawia002** [karmada-io/dashboard#120]
**What happened**:

<img width="522" alt="Screenshot 2024-09-12 at 09 53 44" src="https://github.com/user-attachments/assets/2d64d277-6599-42fa-94c2-dd9d4bb6bbc4">
<img width="596" alt="Screenshot 2024-09-11 at 16 35 53" src="https://github.com/user-attachments/assets/730509fd-a4d2-4810-8903-930ee9aab60d">

After successfully deploying Karmada and the dashboard, and logging into the dashboard successfully, when redirecting to the overview page, the page crashes immediately. All components are running normally, and the API returns responses as expected:

<img width="636" alt="Screenshot 2024-09-12 at 09 53 38" src="https://github.com/user-attachments/assets/85cb8a15-98e7-4c43-bf2e-0834405c955d">

**How to reproduce it (as minimally and precisely as possible)**:

The issue should be reproducible in a newly deployed environment.

**Anything else we need to know?**:

**Environment**:
- Karmada version: 1.11
- Karmada Dashboard version: latest

/cc @warjiang

**Comment by warjiang** [comment]
@iawia002 Can you provide the http response of `overview` api, it will be helpful for me to solve the problem.

**Comment by RainbowMango** [comment]
Hi, @iawia002. I appreciate you for spotting this!

If you want to join us in the dashboard development, It would be great!

**Comment by iawia002** [comment]
> @iawia002 Can you provide the http response of `overview` api, it will be helpful for me to solve the problem.

```json
{
    "code": 200,
    "message": "success",
    "data": {
        "karmadaInfo": {
            "version": {
                "gitVersion": "v1.11.0",
                "gitCommit": "e579560af57f9d370295b94b3a33b102f464413a",
                "gitTreeState": "clean",
                "buildDate": "2024-08-31T12:08:03Z",
                "goVersion": "go1.22.6",
                "compiler": "gc",
                "platform": "linux/amd64"
            },
            "status": "running",
            "createTime": "2024-09-11T06:39:19Z"
        },
        "memberClusterStatus": {
            "nodeSummary": {
                "totalNum": 2,
                "readyNum": 2
            },
            "cpuSummary": {
                "totalCPU": 8,
                "allocatedCPU": 4.022222222222222
            },
            "memorySummary": {
                "totalMemory": 2032247552,
                "allocatedMemory": 781294282.4160941
            },
            "podSummary": {
                "totalPod": 220,
                "allocatedPod": 47
            }
        },
        "clusterResourceStatus": {
            "propagationPolicyNum": 0,
            "overridePolicyNum": 0,
            "namespaceNum": 8,
            "workloadNum": 0,
            "serviceNum": 3,
            "configNum": 9
        }
    }
}
```

**Comment by warjiang** [comment]
@iawia002 Can you describe the karmada-dashboard-web pod and show me the sha256 of the image
![image](https://github.com/user-attachments/assets/c1130cf1-b115-4966-8849-1ebeebd84915)

BTW,  It is probably a problem with react-router and i18n.

**Comment by iawia002** [comment]
docker.io/karmada/karmada-dashboard-web@sha256:066b4ceaf1be713cf6a40ada20c292dc0b88675fe107c42f304e5e3e57eb44f8

**Comment by warjiang** [comment]
@iawia002 I tried to replace the image as you metioned, but still cannot reproduced the problem,  I want to make an online meeting(20 miniutes est.) for more details if it's ok.

**Comment by iawia002** [comment]
Sure, email me the meeting link `z2d at jifangcheng.com`

**Comment by iawia002** [comment]
After some online debugging with @warjiang, we confirmed that the issue was caused by a missing [configuration file](https://github.com/karmada-io/dashboard/blob/main/artifacts/dashboard/karmada-dashboard-configmap.yaml). The dashboard's Helm charts were not updated in time, which led to the problem when installing the dashboard via Helm chart. I will submit a patch later to fix the issue in the chart.
